core/src/repair_weighted_traversal.rs

```diff
@@ -216,6 +227,11 @@ pub mod test {
         );
     }
 
+    fn post_shred_deferment_timestamp() -> u64 {
```

Comment:
I'm thinking we should be able to call into the other shred deferment function if we open up access and eliminate this duplication

Reply:
done